build.gradle.kts

```diff
@@ -230,6 +230,8 @@ dependencies {
     testImplementation("org.testcontainers:toxiproxy:1.16.2")
     testImplementation("org.apache.poi:poi-ooxml:5.1.0")
     testImplementation("com.zaxxer:HikariCP:4.0.3")
+    testImplementation("software.amazon.awssdk:secretsmanager:2.17.189")
+    testImplementation("com.amazonaws.secretsmanager:aws-secretsmanager-caching-java:1.0.2")
```

Comment:
This cache uses Secret Manager ver.1.x :( 
We need either to upgrade this cache to support ver.2.x, or to replace it with our own solution.